deepchem/models/tensorgraph/layers.py

```diff
@@ -2489,7 +2489,6 @@ def __init__(self, batch_size, activation_fn=None, **kwargs):
     super(GraphGather, self).__init__(**kwargs)
 
   def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
-    with tf.device('/cpu'):
       inputs = self._get_input_tensors(in_layers)
```

Comment:
You will have to un-indent things that were in this indentation block.

Reply:
Done